### PR TITLE
new_env/laa-ccms-staging-file-upload-api-production

### PR DESCRIPTION
**Pull request by EricH-MOJDigital**
new_env/laa-ccms-staging-file-upload-api-production

### CODE REVIEW

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-ccms-staging-file-upload-api-production/00-namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: "laa-ccms-staging-file-upload-api-production"
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
```
Comment (rajindersinghsandhu) [inline namespaces/live.cloud-platform.service.justice.gov.uk/laa-ccms-staging-file-upload-api-production/00-namespace.yaml:6]: This should be true

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-ccms-staging-file-upload-api-production/resources/ecr.tf [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  repo_name = "${var.namespace}-ecr"
+  namespace = var.namespace
+
+  # OpenID Connect configuration
+  oidc_providers        = ["github"]
+  github_repositories   = ["laa-ccms-staging-file-upload-api"]
+  github_actions_prefix = "production"
+
+  # OPTIONAL: GitHub environments, to create variables as actions variables in your environments
+  github_environments = ["production"]
```
Comment (rajindersinghsandhu) [inline namespaces/live.cloud-platform.service.justice.gov.uk/laa-ccms-staging-file-upload-api-production/resources/ecr.tf:21]: Double check if we have **production** environment in github repo.

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-ccms-staging-file-upload-api-production/resources/serviceaccount.tf [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  github_actions_secret_kube_token     = var.github_actions_secret_kube_token
+  github_actions_secret_kube_cluster   = var.github_actions_secret_kube_cluster
+  github_actions_secret_kube_namespace = var.github_actions_secret_kube_namespace
+  serviceaccount_token_rotated_date    = "01-01-2000"
+
+  # Uncomment and provide repository names to create github actions secrets
+  # containing the ca.crt and token for use in github actions CI/CD pipelines
+  # github_repositories = ["my-repo"]
+  github_repositories = ["laa-ccms-staging-file-upload-api"]
+  github_environments = ["production"]
```
Comment (rajindersinghsandhu) [inline namespaces/live.cloud-platform.service.justice.gov.uk/laa-ccms-staging-file-upload-api-production/resources/serviceaccount.tf:17]: same here, checking production env in github

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-ccms-staging-file-upload-api-production/resources/ecr.tf (hunk 2 above)
Comment (EricH-MOJDigital) [reply namespaces/live.cloud-platform.service.justice.gov.uk/laa-ccms-staging-file-upload-api-production/resources/ecr.tf:21]: yes we have a prod and production env

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-ccms-staging-file-upload-api-production/resources/serviceaccount.tf (hunk 3 above)
Comment (EricH-MOJDigital) [reply namespaces/live.cloud-platform.service.justice.gov.uk/laa-ccms-staging-file-upload-api-production/resources/serviceaccount.tf:17]: yes we have a prod and production env

**Pull request merged**